### PR TITLE
docs: Update faq.rst

### PR DESCRIPTION
**Pull request by sillyElvy**
## About

This PR updates the faq

## Checklist

- [ ] I've ran `pre-commit` to format and lint the change(s) made.
- [ ] I've checked to make sure the change(s) work on `3.8.6` and higher.
- [ ] This fixes/solves an [Issue](https://github.com/goverfl0w/discord-interactions/issues).
  - (If existent):
- [ ] I've made this pull request for/as: (check all that apply)
  - [x] Documentation
  - [ ] Breaking change
  - [ ] New feature/enhancement
  - [ ] Bugfix

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -140,6 +148,38 @@ can take a page out of discord.js' book if you want to do this, since they've ne
...
+A list of all official extensions can be found `on our github page`_
+
+Those are (at the moment):
+
+- A `voice client`_ - which is still WIP, but it is able to listen for the ``VOICE_STATE_UPDATE`` event.
+- An `autosharder`_ for automatic sharding
+- `wait_for`_ allows listening and waiting for a specific event or a component inside a function
+- `files`_ for file sending with ctx
+- `Molter`_ for message commands
+- `enhanced`_ what enhances the DX in general
```
Comment (Toricane) [inline docs/faq.rst]:
```suggestion
- `enhanced`_ which enhances the DX in general
```

File: docs/faq.rst [hunk 2]
```diff
@@ -165,13 +205,118 @@ At the moment the Discord API does *not* include them into their responses.
...
+channel by the API. The only possible thing is to install the `voice client`_ extension and listen for the
+``VOICE_STATE_UPDATE`` event and then having a local cache that gets updated when a member joins or leaves.
+
+
 My question is not answered on here!
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Please join our Discord server for any further support regarding our library and/or any integration code depending on it.
-
-* Invite Link: https://discord.gg/KkgMBVuEkx
+Please join our `Discord server`_ for any further support regarding our library and/or any integration code depending on it.
```
Comment (Toricane) [inline docs/faq.rst]:
```suggestion
Please join our `Discord Server`_ for any further support regarding our library and/or any integration code depending on it.
```
Consistency

File: docs/faq.rst [hunk 3]
```diff
@@ -7,18 +7,15 @@ Got a question about our library? Well, your answer is probably laying somewhere
 
     This page is maintained by the Helpers of the Discord server,
     and developers at their discretion. For any
-    comments, feedback or concerns please consider joining our server
+    comments, feedback or concerns please consider joining our `Discord Server`_
```
Comment (Toricane) [inline docs/faq.rst]:
```suggestion
    comments, feedback or concerns, please consider joining our `Discord Server`_
```

File: docs/faq.rst [hunk 4]
```diff
@@ -7,18 +7,15 @@ Got a question about our library? Well, your answer is probably laying somewhere
...
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The short answer is: **no**.
 
-The decision to become a standalone library that is now an API wrapper for the Discord API
-was made before Danny posted his gist on GitHub about ceasing development of discord.py.
-This is the official statement from the library developer regarding this:
+The current state of discord.py and its further development is very unclear. What about your lib?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+We have no plans on stopping development and will be available for you. If you have any questions about the
+internal development feel free to ask in the discord support server.
```
Comment (Toricane) [inline docs/faq.rst]:
```suggestion
internal development, feel free to ask in the Discord support server.
```

File: docs/faq.rst [hunk 5]
```diff
@@ -7,18 +7,15 @@ Got a question about our library? Well, your answer is probably laying somewhere
...
     and bringing it up in our support channels.
 
-discord.py is dead! Will this library die too?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The short answer is: **no**.
 
-The decision to become a standalone library that is now an API wrapper for the Discord API
-was made before Danny posted his gist on GitHub about ceasing development of discord.py.
-This is the official statement from the library developer regarding this:
+The current state of discord.py and its further development is very unclear. What about your lib?
```
Comment (Toricane) [inline docs/faq.rst]:
```suggestion
The current state of discord.py and its further development is unclear. What about your library?
```

File: docs/faq.rst [hunk 6]
```diff
@@ -140,6 +148,38 @@ can take a page out of discord.js' book if you want to do this, since they've ne
...
+- `wait_for`_ allows listening and waiting for a specific event or a component inside a function
+- `files`_ for file sending with ctx
+- `Molter`_ for message commands
+- `enhanced`_ what enhances the DX in general
+- A `paginator`_ for paginating embeds on messages using components
+- `persistence`_ - for storing data inside your custom IDs (as an alternative to ``wait_for``)
+- `rest`_ for connecting to a ``RESTful API``
+- And a `boilerplate`_
+
+Below are a few not yet officially supported exts what allowing some functional that are similar to what d.py had:
```
Comment (Toricane) [inline docs/faq.rst]:
```suggestion
Below are a few unofficial exts (for the time being) which implement some functionality similar to what d.py had:
```

File: docs/faq.rst [hunk 7]
```diff
@@ -140,6 +148,38 @@ can take a page out of discord.js' book if you want to do this, since they've ne
...
+- `checks and cooldowns`_
+- `tasks`_
+- `get`_ for getting objects from the discord API (will be implemented into the core library at a later time)
+
+Usage examples can usually be found at the linked page
+
+
+Can I make an Extension Library myself?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Yeah you can! We have a special channel reserved for the development of external libraries!
```
Comment (Toricane) [inline docs/faq.rst]:
```suggestion
Yeah, you can! We have a special channel reserved for the development of external libraries!
```

**Review by Toricane — CHANGES_REQUESTED** [review]
Some grammar suggestions
Also why are there deleted sections? That's not related to this PR

**Comment by sillyElvy** [comment]
> Some grammar suggestions
> Also why are there deleted sections? That's not related to this PR 

This PR is about to update the FAQ section. So it is definitely related imo. I deleted those sections since they origin from way before when Danny announced his end of development, but I don't feel like those questions are *Frequently Asked* anymore



Ok before sending this comment I saw wym, I have to merge unstable into my branch ig

**Comment by sillyElvy** [comment]
> > Some grammar suggestions
> > Also why are there deleted sections? That's not related to this PR
> 
> This PR is about to update the FAQ section. So it is definitely related imo. I deleted those sections since they origin from way before when Danny announced his end of development, but I don't feel like those questions are _Frequently Asked_ anymore
> 
> Ok before sending this comment I saw wym, I have to merge unstable into my branch ig

Ok on gh in the browser there are actually no deleted sections outside the faq lol

**Review by Toricane — APPROVED** [review]
LGTM!

**Pull request merged**